pkg/auth/auth.go

```diff
@@ -321,79 +318,59 @@ func AuthFrontend(cl client.Client, rule *config.FrontendRule) error {
 		err = cl.Send(&pgproto3.AuthenticationSASLFinal{Data: []byte(finalMsg)})
 		return err
 	case config.AuthLDAP:
-		if rule.AuthRule.LDAPConfig == nil {
```

Comment:
Why did you remove this check?

Reply:
I forgot to add it back when I removed previous code. Fixed